cuebot/src/main/java/com/imageworks/spcue/dao/postgres/ProcDaoJdbc.java

```diff
@@ -264,30 +264,25 @@ public void updateProcMemoryUsage(FrameInterface f, long rss, long maxRss,
                     "SELECT pk_frame FROM proc WHERE pk_frame=? FOR UPDATE",
                     String.class, f.getFrameId()).equals(f.getFrameId())) {
 
-                getJdbcTemplate().update(UPDATE_PROC_MEMORY_USAGE,
```

Comment:
I'm pretty sure this was a merge mistake. This block and the one below it seem to duplicate each other, and this block is missing one of the placeholder values.